packages/core/src/Stores/Modules/Cashier/on-ramp-store.js

```diff
@@ -33,9 +33,21 @@ export default class OnRampStore extends BaseStore {
                         this.default_from_currency
                     }&toDefault=${currency}&theme=danger&merchant_id=iiq3jdt2p44yrfbx&payment_id=&v=2" width="100%" height="475px" class="changelly" scrolling="no" onLoad="function at(t){var e=t.target,i=e.parentNode,n=e.contentWindow,r=function(){return n.postMessage({width:i.offsetWidth},it.url)};window.addEventListener('resize',r),r()};at.apply(this, arguments);" style="min-height: 100%; min-width: 100%; overflow-y: visible; border: none">Can't load widget</iframe>`;
                 },
-                icon: 'IcCashierChangelly',
+                icon: isMobile() ? 'IcCashierChangellyMobile' : 'IcCashierChangelly',
```

Comment:
Could we use `is_mobile` from `ui-store.js` instead? This will ensure component gets re-rendered when resizing screen.

Reply:
it's not inside a component and it seems OK to use isMobile() here